src/pages/login-page.tsx

```diff
@@ -64,19 +64,13 @@ const LoginPage = (props: PageProps<void>) => {
       <Text style={styles.title}>Welcome to Taqtile!</Text>
       <View style={styles.viewLogin}>
         <Text style={styles.textLogin}>E-mail</Text>
-        <TextInput
-          style={styles.inputLogin}
-          autoCapitalize="none"
-          onChangeText={(text) => setEmail(text)}>
-          {email}
-        </TextInput>
+        <Input autoCapitalize="none" onChangeText={(text) => setEmail(text)} />
```

Comment:
qdo vc só repassa o mesmo contexto (text: string), vc pode simplificar:
```suggestion
        <Input autoCapitalize="none" onChangeText={setEmail} />
```

Reply:
O mesmo vale para useRef? se eu colocar onChangeText={email.current} ele entende?